modules/test/playwright/pages/commerce/commerceProductAdminPage.ts

```diff
@@ -37,4 +103,16 @@ export class CommerceProductAdminPage {
 		await this.creationMenuNewButton.click();
 		await this.generateSkusMenuItem.click();
 	}
+
+	async goto() {
+		await this.applicationsMenuPage.goToProducts();
+	}
+
+	async goToProductRelations() {
+		await this.productRelationsLink.click();
+	}
```

Comment:
I think a lot of these new "elements" are to manage the content of a tab inside the product page? If is correct could we create a new page to manage the specific tab behavior? Like commerceProductAdminProductRelationshipPage.ts ... in this way we will have them separated. Otherwise when we will have all tabs this page will be a very confused and difficult to manage